s3/pkg/datastore/driver/clean.go

```diff
@@ -0,0 +1,17 @@
+package driver
+
+type Closer interface {
+	Close()
+}
+
+var closers []Closer
+
+func AddCloser(closer Closer) {
+	closers = append(closers, closer)
```

Comment:
can there be multiple instances of the closer type in the slice ?

Reply:
Hi rhsakarpos, sorry, could you please clarify the meaning of 'multiple instances of a closer type'?